Sources/Common/Background Queue/QueueRunner.swift

```diff
@@ -66,12 +79,6 @@ private extension CioQueueRunner {
             return
         }
 
-        let httpParams = HttpRequestParams(
-            endpoint: .trackDeliveryMetrics,
-            headers: nil,
-            body: bodyData
-        )
-
-        performHttpRequest(params: httpParams, onComplete: onComplete)
+        performHttpRequest(endpoint: .trackDeliveryMetrics, requestBody: bodyData, onComplete: onComplete)
```

Comment:
This is why the changes in `ApiSyncQueueRunner` exist. Encapsulate the construction of `HttpRequestParams` in 1 place. 

It would be nice if we didn't even need these queuerunner classes as they are heavy in boilerplate. But, we can try to do that in KMM instead of a refactor to this SDK.